src/routes/safe/components/Transactions/TxList/hooks/useActionButtonsHandlers.ts

```diff
@@ -48,7 +52,17 @@ export const useActionButtonsHandlers = (transaction: Transaction): ActionButton
           (canExecute && details.confirmationsRequired > details.confirmations.length) ||
           (canConfirmThenExecute && details.confirmationsRequired - 1 > details.confirmations.length)
         ) {
-          dispatch(showNotification(NOTIFICATIONS.TX_FETCH_SIGNATURES_ERROR_MSG))
+          const deeplink = generatePath(SAFE_ROUTES.TRANSACTIONS_SINGULAR, {
+            [SAFE_ADDRESS_SLUG]: getPrefixedSafeAddressSlug({ shortName, safeAddress }),
+            [TRANSACTION_ID_SLUG]: transaction.id,
+          })
+
+          dispatch(
+            showNotification({
+              ...NOTIFICATIONS.TX_FETCH_SIGNATURES_ERROR_MSG,
+              link: { to: deeplink, title: 'View Transaction' },
```

Comment:
`View transaction`

Reply:
All other notification `title`s follow this style.